### ISSUE TITLE
"Use Rosetta for x86/AMD64 emulation on Apple Silicon" not available 

### ISSUE THREAD

**Issue by hellozyemlya** [docker/for-mac#6788]
- [x] I have tried with the latest version of Docker Desktop
  - [ ] I have tried disabling enabled experimental features
  - [ ] I have uploaded Diagnostics
  - Diagnostics ID:

### Expected behavior
I want to see "Use Rosetta for x86/AMD64 emulation on Apple Silicon" checkbox.
### Actual behavior
There is no this checkbox.
### Information

  - macOS Version: 12.6.1
  - Intel chip or Apple chip: M1
  - Docker Desktop Version: 4.17.0

### Output of `/Applications/Docker.app/Contents/MacOS/com.docker.diagnose check`
[PASS] DD0027: is there available disk space on the host?
[PASS] DD0028: is there available VM disk space?
[PASS] DD0018: does the host support virtualization?
[PASS] DD0001: is the application running?
[PASS] DD0017: can a VM be started?
[PASS] DD0016: is the LinuxKit VM running?
[PASS] DD0011: are the LinuxKit services running?
[PASS] DD0004: is the Docker engine running?
[PASS] DD0015: are the binary symlinks installed?
[PASS] DD0031: does the Docker API work?
[PASS] DD0013: is the $PATH ok?
[PASS] DD0003: is the Docker CLI working?
[PASS] DD0038: is the connection to Docker working?
[PASS] DD0014: are the backend processes running?
[PASS] DD0007: is the backend responding?
[PASS] DD0008: is the native API responding?
[PASS] DD0009: is the vpnkit API responding?
[PASS] DD0010: is the Docker API proxy responding?
[SKIP] DD0030: is the image access management authorized?
[PASS] DD0033: does the host have Internet access?
[PASS] DD0018: does the host support virtualization?
[PASS] DD0001: is the application running?
[PASS] DD0017: can a VM be started?
[PASS] DD0016: is the LinuxKit VM running?
[PASS] DD0011: are the LinuxKit services running?
[PASS] DD0004: is the Docker engine running?
[PASS] DD0015: are the binary symlinks installed?
[PASS] DD0031: does the Docker API work?
[PASS] DD0032: do Docker networks overlap with host IPs?
No fatal errors detected.

**Comment by sadjadteh** [comment]
Having the same issue with:

- macOS Version: 12.5
- Intel chip or Apple chip: M1
- Docker Desktop Version: 4.17.0

**Comment by AbdullahWali** [comment]
> macOS Version: 12.6.1
This feature is only on Ventura (macOS 13) i believe

**Comment by sadjadteh** [comment]
> > macOS Version: 12.6.1
> > This feature is only on Ventura (macOS 13) i believe

You're right. I updated the macOS version and I can see the "Use Rosetta for x86/AMD64 emulation on Apple Silicon" checkbox now. Thanks a lot.
